resources/todomvc/architecture-examples/svelte-complex/package-lock.json

```diff
@@ -10,7 +10,6 @@
             "devDependencies": {
                 "big-dom-generator": "file:../../big-dom-generator",
                 "http-server": "^14.1.1",
-                "jsdom": "^22.1.0",
```

Comment:
@lpardosixtosMs - this change happened when I ran npm install, to be able to rebuild the new svelte version.

Reply:
That is okay. We've removed jsdom as a devDependency in package.json. I was using npm ci which doesn't update the lock files. This change is okay though.

Reply:
This makes me think that we probably want `npm install` and not `npm ci` when updating the complex versions. Indeed we might run into problems when adding dependencies to the big-dom-generator. It would be good to double check.